tests/django_restframework_gis_tests/test_schema_generation.py

```diff
@@ -608,3 +609,14 @@ def test_distance_to_point_filter(self):
                 },
             ],
         )
+
+    def test_geometry_field(self):
+        path = "/"
+        method = "GET"
+        view = create_view(ModelViewWithPolygon, "GET", create_request("/"))
+        inspector = GeoFeatureAutoSchema()
+        inspector.view = view
+        serializer = inspector.get_serializer(path, method)
+        content = inspector.map_serializer(serializer)
+
+        print(content)
```

Comment:
can we compare `content` against the expected value?

Reply:
Absolutely haha. That's my bad, I had uncommitted changes before pushign